project/README.md

```diff
@@ -15,7 +15,20 @@
 
 ## React
 
-## Swift
+## Swift   
+[Tanay Khandke](https://github.com/tkhandke)
+
+**Major**
+
+* Documentation -   
+tips for swift unit testing on linux - https://github.com/tkhandke/swift_doc
```

Comment:
Did you make a pull request for the changes you made to the documentation? We want to contribute to the project.